ISISComputingGroup/ibex_gui
# Ticket1936: synoptic PV address validation in title

### Description of work

Added the PV validation to the title bar rather than a separate label. See https://github.com/ISISComputingGroup/IBEX/issues/1936

Also merge: https://github.com/ISISComputingGroup/ibex_system_tests/pull/51

### To test

* Confirm new unit tests run
* Confirm that creating a PV with a bad address causes an error on the title bar
* Confirm this causes you to be unable to save
* Confirm that this persists when selecting new components
* Confirm that the component and pv name in the title bar get updated properly

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
